docker/0.5.0/fedimintd-mainnet-tls/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "<federation_id_hex>": {
+    "federation_name": "Override Federation Name"
+  }
+}
```

Comment:
Do we still want this with 0.5? I guess for 0.3 client compat?

Reply:
@elsirion I still need to clean it up properly.